MANIFEST.in

```diff
@@ -0,0 +1,8 @@
+recursive-include ilpy *.cpp *.h *.pxd *.py *.pyx *.txt
```

Comment:
this file makes it so the `sdist` (built by `python setup.py sdist`) includes all the necessary files that someone could actually build the wheel later if they wanted to (assuming they had scip and gurobi installed somehow)